azure-pipelines.yml

```diff
@@ -137,6 +143,10 @@ jobs:
         inputs:
           script: 'dotnet build $(PathToCommunityToolkitSampleCsproj) -c Release'
       - task: CmdLine@2
-        displayName: 'Pack CommunityToolkit NuGets'
+        displayName: 'Pack CommunityToolkit.Maui.Core NuGet'
+        inputs:
+          script: 'dotnet pack $(PathToCommunityToolkitCoreCsproj) -c Release -p:PackageVersion=$(NugetPackageVersion) --output $(Build.ArtifactStagingDirectory)/nuget'
+      - task: CmdLine@2
+        displayName: 'Pack CommunityToolkit.Maui NuGet'
         inputs:
-          script: 'dotnet pack $(PathToCommunityToolkitCsproj) -c Release -p:PackageVersion=$(NugetPackageVersion) --output $(Build.ArtifactStagingDirectory)/nuget'
\ No newline at end of file
+          script: 'dotnet pack $(PathToCommunityToolkitCoreCsproj) -c Release -p:PackageVersion=$(NugetPackageVersion) --output $(Build.ArtifactStagingDirectory)/nuget'
```

Comment:
looks like you pack Core multiple times. it should remain PathToCommunityToolkitCsproj

Reply:
Whoops - good catch!

Reply:
Fixed ✅